src/component/container/page_list.tsx

```diff
@@ -11,14 +11,109 @@ export interface PageListProps {
 	store: Store;
 }
 
+export interface PageListItemProps {
+	id: string;
+	name: string;
+	pageRef: PageRef;
+	store: Store;
+}
+
+@observer
+export class PageListItem extends React.Component<PageListItemProps> {
+	@MobX.observable protected pageElementEditable: boolean = false;
+	@MobX.observable protected pageName: string;
```

Comment:
renaming this to `pageNameInputValue` would make it a little more clearer that this value is not actually the page name. just the editable value in the ui that becomes the page name after applying.

Reply:
Done ✅